client/src/components/Navbar/Notification/ReadNotification.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { ReactElement } from 'react';
+import NotificationInterface from '../../../interface/Notification';
+import moment from 'moment';
+import { Divider, ListItemText, ListItem, Avatar, Typography, ListItemAvatar } from '@mui/material';
+import { useStyles } from '../useStyles';
+import { NavLink } from 'react-router-dom';
+
+interface Props {
+  notification: NotificationInterface;
+}
+
+export default function UnreadNotification({ notification }: Props): ReactElement {
```

Comment:
`UnreadNotification` and `ReadNotifications` almost look identical. What is the reasoning behind componentizing this?

Reply:
I made two UI elements to render based on the notification.read prop according to what I thought Ugur said to do ( I know its DRY code) but now I finally figured out how to change styles based on a prop you can pass to use styles :D. This took be me a few tries to figure it out but hopefully this implementation looks better.

Reply:
Ah yes! I had something closer to this in mind:
```
<Notification title={notification.title} body={notification.desc} read={notification.read} ... />
```
Sorry about the confusion here 😅 Usually, componentizing entails refactoring a part of the UI into smaller components, in a modular and reuseable way. So if our advice seems like it doesn't align with these principles, feel free to ask for more clarification! 👍🏼